src/ReportPortal.Shared/Reporter/Http/HttpClientHandlerFactory.cs

```diff
@@ -36,10 +36,18 @@ public virtual HttpClientHandler Create()
 
             httpClientHandler.Proxy = GetProxy();
 
+            var ignoreSslErrors = Configuration.GetValue<bool>("Server:IgnoreSslErrors", true);
+            
 #if NETSTANDARD2_0
```

Comment:
We forgot net6 recently introduced as a target framework. I propose revert if statement to
```
if !net462
  // use httpClientHandler  
else
  // use ServicePointManager
```